contents/articles/2017-01_retina-mapping/index.md

```diff
@@ -0,0 +1,24 @@
+--- 
+title: Retinotopic to Spatiotopic Mapping in Blind Patients Implanted with Retinal Prosthesis
+date: Mon Jan 30 2017 10:36:53 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Blind patients implanted with a retinal prosthesis fitted to a modified Pupil headset to measure the accuracy in mapping stimulated retinal signals to the world coordinates..."
+featured_img: "../../../../media/images/blog/epiretinal-array-dark.png"
+featured_img_thumb: "../../../../media/images/blog/thumb/epiretinal-array.png" 
+---
+
+<div align="center">
+	<img src="../../../../media/images/blog/thumb/epiretinal-array.png" class='Feature-image u-padBottom--1' alt="Epiretinal Array of Electrodes" style="width: 50%;">
+	<div class="small u-padBottom--3">Image Source: [Retinotopic to Spatiotopic mapping in blind patients implated with the Argus II retinal prosthesis](http://iovs.arvojournals.org/article.aspx?articleid=2597840#149774812)
+	</div>
+</div>
+
+The observation that electrical stimulation of retina in blind patients can create visual percept led to the development of retinal prosthesis devices implanted in blind patients. For the percept to be useful, the electrical stimulation should convey information to the brain with the correct spatial coordinates.
+
+In their paper [Retinotopic to Spatiotopic mapping in blind patients implated with the Argus II retinal prosthesis](http://iovs.arvojournals.org/article.aspx?articleid=2597840#149774812), Avi Caspi et al., researched blind patients implanted with a retinal prosthesis fitted to a modified [Pupil headset](https://pupil-labs.com/store/) to measure the accuracy in mapping stimulated retinal signals to the world coordinates. And to demonstrate that an eye tracker can be calibrated on the patient based on the retinal prosthesis.
+
+In the experiment, the patient's gaze positions were recorded with timestamps synched to the onset of the stimulation using [Pupil eye tracker](https://pupil-labs.com/store/), and by indicating the location of the percepts using a handheld marker recorded with the scene camera. 
```

Comment:
When using Pupil, for example, a sighted individual typically looks at a marker that can be detected by the world camera to calibrate. However, "with blind subjects, calibration of eye tracking is not a trivial matter."  The researchers developed a calibration procedure by activating specific clusters of electrodes in the retinal implant. For each cluster activated, the researchers then asked the participants to hold a red ball at arm's length and align the red ball with the location of the "light". The red ball was used as a marker that could be tracked in the space of the world camera. The researchers then know the position of the pupil, the location of the stimulus on the retina, and the position of the marker within the world camera space. With these known coordinates the researchers are able to solve a system of equations to establish a 2d mapping from pupil coordinates to gaze coordinates.